source/ImageBuffer.cpp

```diff
@@ -314,11 +312,28 @@ namespace {
 			rows[y] = reinterpret_cast<JSAMPLE *>(buffer.Begin(y, frame));
 		
 		while(height)
+			// starts each line at corresponding rows[y] location, each line using
+			// only 3/4 of the space allocated for the line by ImageBuffer
 			height -= jpeg_read_scanlines(&cinfo, &rows.front() + cinfo.output_scanline, height);
 		
 		jpeg_finish_decompress(&cinfo);
 		jpeg_destroy_decompress(&cinfo);
 		
+		// Expand RGB to RGBA
+		for (unsigned int y = 0; y < cinfo.image_height; y++)
+		{
+			auto pptr = reinterpret_cast<uint8_t*>(buffer.Begin(y, frame));
+			// expands the 3 byte RBG values in a line into 4-byte RGBA values
+			// using up what was empty space at the end of each line
+			for(int idx = width - 1; idx >= 0; --idx)
```

Comment:
Do we also have an idea of how much performance costs (at loading-time) we have because of the additional loop that is now needed? (Or performance gain, but that seems a bit unlikely.)

Reply:
Performance gain seems possible! average 5% faster, not significantly different with 9 observations each. https://observablehq.com/@ballingt/timing-data-for-endlesssky-5153